x/rewards/keeper/flat_fee.go

```diff
@@ -37,3 +40,36 @@ func (k Keeper) GetFlatFee(ctx sdk.Context, contractAddr sdk.AccAddress) (sdk.Co
 
 	return fee, true
 }
+
+// ImportFlatFees initializes state from the genesis flat fees data.
+func (k Keeper) ImportFlatFees(ctx sdk.Context, flatFees []types.FlatFee) {
+	for _, flatFee := range flatFees {
+		if err := k.SetFlatFee(ctx, flatFee.MustGetContractAddress(), flatFee.GetFlatFee()); err != nil {
+			panic(fmt.Sprintf("flat fee: %+v is invalid: %s", flatFee, err))
+		}
+	}
+}
+
+// ExportFlatFees returns the flat fees genesis data for the state.
+func (k Keeper) ExportFlatFees(ctx sdk.Context) []types.FlatFee {
```

Comment:
Would be better to implement this in `state_flatfee.go` to maintain consistency.

Reply:
Will need to pass metadata state as argument if we move to `state_flatfee.go` right? Or am I missing something here?

Reply:
@spoo-bar Have refactored as mentioned above ^^ (a365d76). Let me know if this is fine or if we can do something better here.